### ISSUE TITLE
Bitcoins lost

### ISSUE THREAD

**Issue by bvd1919** [bitpay/copay#6057]
I sent bitcoins from my blockchain wallet to my Copay address. First payment went fine, second payment won't show up, but are taken from my blockchain wallet. Where are my bitcoins and how can I get them back?

Thanks in advance.

**Comment by dabura667** [comment]
Transaction IDs please.

**Comment by bvd1919** [comment]
Working transaction: https://blockchain.info/tx/fbef75b6c9414aa421c812afa8354bd45a04cd96cce7e197daf0afef057f1a2f

Not working transaction:

**Comment by bvd1919** [comment]
Sorry something went wrong while posting.

Working transaction: https://blockchain.info/tx/fbef75b6c9414aa421c812afa8354bd45a04cd96cce7e197daf0afef057f1a2f

Not working transaction: https://blockchain.info/tx/4c5b22ec62aa7dc71022b05b285884fcb39cc7e11e0f5844b05b98baeb1b1f3f

**Comment by dabura667** [comment]
The second transaction is dependent on an unconfirmed parent transaction.

tl;dr wait and it will confirm eventually. Once it confirms, your wallet should show it.
